File: solutions/2053-check-if-all-characters-have-equal-number-of-occurrences/solution.py
class Solution(object):
    def areOccurrencesEqual(self, s):
        """
        :type s: str
        :rtype: bool
        """
        st=set(list(s))
        ls=[s.count(i) for i in st]
        for i in ls:
            if i!=ls[0]:
                return False
        return True

File: solutions/2053-check-if-all-characters-have-equal-number-of-occurrences/solution.java
class Solution {
    public boolean areOccurrencesEqual(String s) {
        Set<Character>ls=new HashSet<>();
        List<Character>ls2=new ArrayList<>();
        for(char a:s.toCharArray())
        {
            ls.add(a);
            ls2.add(a);
        }
        int n=Collections.frequency(ls2,s.charAt(0));
        for(char i:ls)
            if(Collections.frequency(ls2,i)!=n)
                return false;
        return true;
    }
}
